### PR TITLE
Update search-the-audit-log-in-security-and-compliance.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the export details as users can download only 50000 results

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Here's the process for searching the audit log in Microsoft 365.
 
 - Click **Clear** to clear the current search criteria. The date range returns to the default of the last seven days. You can also click **Clear all to show results for all activities** to cancel all selected activities.
 
-- If 50,000 results are found, you can probably assume that there are more than 50,000 events that met the search criteria. You can either refine the search criteria and rerun the search to return fewer results, or you can export all of the search results by selecting **Export results** \> **Download all results**.
+- If 50,000 results are found, you can probably assume that there are more than 50,000 events that met the search criteria. You can either refine the search criteria and rerun the search to return fewer results, or you can export all of the 50,000 search results by selecting **Export results** \> **Download all results**.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md]:
```suggestion
- If 50,000 results are found, you can probably assume that there are more than 50,000 events that met the search criteria. You can either refine the search criteria and rerun the search to return fewer results, or you can export the 50,000 search results by selecting **Export results** \> **Download all results**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**